content/php/concepts/arrays/terms/array-chunk/array-chunk.md

```diff
@@ -0,0 +1,122 @@
+---
+Title: 'array_chunk()'
+Desciprtion: 'Splits an array into chunks'
```

Comment:
```suggestion
Description: 'Splits an array into a number of specified chunks.'
```

Reply:
the spacing within the metadata section looks incorrect. Please look at one of the templates or an existing doc for reference